lib/scan/cms_detection/engine.py

```diff
@@ -23,7 +23,8 @@
 from core._time import now
 from core.log import __log_into_file
 import requests
-from lib.http_fuzzer.engine import user_agents_list
+from lib.payload.wordlists import useragents
```

Comment:
@securestep9  First we need to push #272 in order to successfully import useragents :)